src/database/src/lib.rs

```diff
@@ -119,7 +119,7 @@ pub fn get_conn_devel() -> Result<crate::DbConnection, diesel::result::Connectio
                 "postgres://postgres:dimpostgres@postgres/dim_devel",
             )?;
         } else {
-            let mut conn = DbConnection::establish("./dim_devel.db;foreign_keys=true;")?;
+            let conn = DbConnection::establish("./dim_devel.db;foreign_keys=true;")?;
```

Comment:
can you remove the foreign_keys=true thingie including the `;`

Reply:
```suggestion
            let conn = DbConnection::establish("./dim_devel.db;")?;
```